client/components/Help/Help.js

```diff
@@ -68,4 +107,15 @@ const mapStateToProps = ({ problems }) => {
     };
 };
 
-export default connect(mapStateToProps)(Help);
+const mapDispatchToProps = (dispatch) => {
```

Comment:
1 Important and simple rule: if your component is wrapped in connect - it is container, and should be in containers folder.